roles/core/hosts_file/templates/hosts.j2

```diff
@@ -1,12 +1,11 @@
 #jinja2: lstrip_blocks: "True"
-{# Hosts file, Accelerated mode compatible #}
+{# Hosts file, with direct_access #}
 {# Benoit Leveugle 2020 #}
 {% import "macros/network.j2" as macro_network with context %}
-{% import "macros/accelerated.j2" as macro_accelerated with context %}
+{% import "macros/groups.j2" as macro_groups with context %}
 
-{# Macro to avoid redundancy #}
-{# This macro should cover all technical part, and be compatible with both normal and accelerated mode #}
-{% macro write_host_3(host,host_dict,host_node_main_network_interface,host_iceberg) %}
+{# -- Macro shared by both standard and direct_access methods. -- #}
+{% macro write_host(host,host_dict,host_node_main_network_interface,host_iceberg) %}
 
   {#- Write host #}
   {% if host_dict['network_interfaces'] is defined and not none and host_dict['network_interfaces'] and host_dict['network_interfaces'] is iterable %}
```

Comment:
please remove "and not none" or replace this by either :

`host_dict['network_interfaces'] is defined and host_dict['network_interfaces'] is not none`

OR 

`(host_dict['network_interfaces'] | default(none)) is not none`

I think variables cannot be set to "none" or left empty (for initialization purpose ?) and should not be checked.

Reply:
```suggestion
  {% if host_dict['network_interfaces'] is defined and host_dict['network_interfaces'] is iterable %}
```
Should be enough?